composer.json

```
@@ -70,7 +71,20 @@
         "bin/sabredav",
         "bin/naturalselection"
     ],
-    "config" : {
-        "bin-dir" : "./bin"
+    "scripts": {
+        "phpstan": [
+            "@php vendor/bin/phpstan analyse lib tests"
```

This should also work?

```suggestion
            "phpstan analyse lib tests"
```

---

Works. I simplified each of `phpstan` `cs-fixer` and `phpunit`
These need to be simplified in `skel` and the change applied across all repos.